frontend/apps/reader/modules/readerfont.lua

```diff
@@ -436,8 +436,8 @@ local FONT_FAMILIES = {
     -- On 2nd page
     { "cursive", _("Cursive") },
     { "fantasy", _("Fantasy") },
-    { "emoji", _("Emoji \u{1F60A}") },
-    { "fangsong", _("Fang Song \u{4EFF}\u{5B8B}") },
+    { "emoji", _("Emoji \xF0\x9F\x98\x8A") }, -- U+1F60A
+    { "fangsong", _("Fang Song \xE4\xBB\xBF\xE5\xAE\x8B") }, -- U+4EFF U+5B8B
```

Comment:
I'm sure some translator would remove these Chinese chars if he sees them :)
Would need a `@translator(s?)` note.

Reply:
That did cross my mind; in this very specific example, I was wondering if we couldn't simply get away with `_("Thing")  .. " U+whatever"` to avoid having this seep through the TLs?